model/DeliveryContainerService.php

```diff
@@ -177,4 +169,44 @@ protected function getActiveFeatures(\core_kernel_classes_Resource $delivery)
             $delivery->getOnePropertyValue($this->getProperty(self::TEST_RUNNER_FEATURES_PROPERTY))
         );
     }
+
+    /**
+     * @param core_kernel_classes_Resource $delivery
+     * @return array
+     */
+    protected function getPluginsDisabledForDelivery(core_kernel_classes_Resource $delivery)
+    {
+        $enabledPlugins = [[]];
+        $disabledPlugins = [[]];
```

Comment:
Just in order to avoid confusion, I would've gone with a new set of variables here.

Reply:
Done